watson/utils.py

```diff
@@ -85,6 +85,11 @@ def _style_short_id(id):
     fmt = formats.get(name, {})
 
     if isinstance(fmt, dict):
+        try:
+            config = click.get_current_context().obj.config
+            fmt.update(config.getitems('style:%s' % name))
+        except (AttributeError, RuntimeError):
+            pass
```

Comment:
I wonder if that could be done somewhere during the startup (maybe in a lazy way?) instead of when styling every element. For long outputs with lots of formatting this might slow things down.

Reply:
@k4nar hi, I added a function to load styles just on the startup:
https://github.com/TailorDev/Watson/pull/335/files#diff-53886fd7d4ba12dfad3dbb6c9716ca57R45-R52

Can you please review it again?